scripts/build/artifactory.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#to be executed from the root project directory
+git rev-parse --short HEAD > version.txt
```

Comment:
> > I think a more reliable solution would be to use full hash instead of its abbreviated version.
> 
> I started with full hash but changed it to shorter version because if we wish to present it somewhere on a web page (e.g. footer) it is less eye catching and easier to fit into page. I think it is still unique and it can be also used with git show command to find the exact commit in log.

It's unique with respect to all previous commit hashes but not in general case. The UI developer could shorten the hash in the UI code if he wants to or hide it somehow.